File: netflipster/java/src/main/java/repository/BookingDao.java
package repository;

import source.DB;

public class BookingDao {

    public static void book(String vhsId, int quantity, String userId) {
        DB.insert(String.format("INSERT INTO BOOKINGS(VHS_ID, QUANTITY, USER_ID) values('%s', %s, '%s')", vhsId, quantity, userId));
    }
}

File: netflipster/csharp/Netflix2/repository/BookingDao.cs
namespace Netflix2.repository
{
    public static class BookingDao
    {
        public static bool Book(string vhsId, int quantity, string userId)
        {
            return DB.Insert($"INSERT INTO BOOKINGS(VHS_ID, QUANTITY, USER_ID) values('{vhsId}', {quantity}, '{userId}')");
        }
    }
}